wasmer2/wasmer2Instance.go

```diff
@@ -224,34 +218,14 @@ func (instance *Wasmer2Instance) ValidateVoidFunction(functionName string) error
 	return nil
 }
 
-// GetData returns a pointer for the current instance's data
-func (instance *Wasmer2Instance) GetData() uintptr {
-	panic("instance GetData")
-}
-
-// GetInstanceCtxMemory returns the memory for the instance context
-func (instance *Wasmer2Instance) GetInstanceCtxMemory() executor.Memory {
-	// return instance.InstanceCtx.Memory()
-	panic("instance GetInstanceCtxMemory")
+// HasMemory checks whether the instance has at least one exported memory.
+func (instance *Wasmer2Instance) HasMemory() bool {
+	return true
```

Comment:
there is no more `instance.memory` in Go. It gets checked in Rust when we create the instance. We no longer keep a separate opaque pointer, we just send the instance each time.

In the future we will get rid of HasMemory. It could have been hidden in Wasmer1, but I reckoned it is not worth the effort.